x/emissions/keeper/genesis.go

```diff
@@ -17,6 +17,9 @@ func (k *Keeper) InitGenesis(ctx context.Context, data *types.GenesisState) erro
 	k.authKeeper.SetModuleAccount(ctx, alloraRewardsModuleAccount)
 	alloraPendingRewardsModuleAccount := k.authKeeper.GetModuleAccount(ctx, types.AlloraPendingRewardForDelegatorAccountName)
 	k.authKeeper.SetModuleAccount(ctx, alloraPendingRewardsModuleAccount)
+	if err := k.SetParams(ctx, data.Params); err != nil {
+		return err
+	}
```

Comment:
so the genesis params weren't actually being set before right?

Reply:
shockingly it appears indeed they were not

it would just always use the defaults and literally ignore what was in the genesis.json